baseapp_payments/serializers.py

```
@@ -272,7 +272,7 @@ def create(self, validated_data):
             )
             return {"id": setup_intent["id"], "client_secret": setup_intent["client_secret"]}
         except Exception as e:
-            logger.error(f"Failed to create setup intent: {str(e)}")
+            logger.exception(f"Failed to create setup intent: {str(e)}")
```

Ruff (0.15.13)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)